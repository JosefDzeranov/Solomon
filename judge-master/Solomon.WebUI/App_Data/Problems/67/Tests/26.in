10
6 3 4
1 2 2
4 3 9
2 7 7
9 1 5
1 2 5
1 6 5
4 6 8
5 5 3
7 3 8
