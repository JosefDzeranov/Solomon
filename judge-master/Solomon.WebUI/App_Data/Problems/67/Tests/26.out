22
3
6 3 10 